Formula/lib/libdap.rb

```diff
@@ -87,6 +63,8 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/dap-config --version")
+    # Versions like `1.2.3-4` simply appear appear as `1.2.3` in the output,
+    # so we have to remove the suffix from the formula version.
+    assert_match version.to_s.sub(/-\d+$/, ""), shell_output("#{bin}/dap-config --version")
```

Comment:
instead of doing suffix trim, why not do 

`assert_match shell_output("#{bin}/dap-config --version"), version.to_s`

Reply:
```
Minitest::Assertion: Expected /libdap\ 3\.21\.0\n/ to match "3.21.0-27
```

The output is `libdap 3.21.0\n` (not just `3.21.0`), so we have to modify the text either way. Your suggestion would require something like `assert_match shell_output("#{bin}/dap-config --version").delete_prefix("libdap ").strip, version` (or `shell_output("#{bin}/dap-config --version").gsub(/^libdap\s*|\s*$/, "")`, which is more involved.

Reply:
make sense 👍